modules/messages/src/lib.rs

```diff
@@ -534,6 +534,7 @@ pub mod pallet {
 	}
 
 	#[pallet::error]
+	#[derive(PartialEq)]
```

Comment:
I could remember I've been fixing [this lint](https://rust-lang.github.io/rust-clippy/master/#/derive_partial_eq_without_eq) some time ago - we could also probably derive the `Eq` here?

Reply:
Done